src/main/java/net/sf/jabref/model/entry/BibtexString.java

```diff
@@ -152,4 +152,28 @@ public String getParsedSerialization() {
     public boolean hasChanged(){
         return hasChanged;
     }
+
+    /*
+    * Returns user comments (arbitrary text before the string) if there are any. If not returns the empty string
+     */
+    public String getUserComments() {
```

Comment:
This is now an exact copy of BibEntry.getUserComments right? Maybe add a superclass BibItem containing this method and then BibEntry and BibString derive from this superclass (maybe there is even more common code which could be extracted to the super class)